src/client.cpp

```diff
@@ -939,7 +940,9 @@ void writePlayerPos(LocalPlayer *myplayer, ClientMap *clientMap, NetworkPacket *
 	u32 keyPressed   = myplayer->keyPressed;
 	// scaled by 80, so that pi can fit into a u8
 	u8 fov           = clientMap->getCameraFov() * 80;
-	u8 wanted_range  = clientMap->getControl().wanted_range / MAP_BLOCKSIZE;
+	int wanted_range = std::ceil(clientMap->getControl().wanted_range / MAP_BLOCKSIZE);
```

Comment:
Actually, `clientMap->getControl().wanted_range` happens to be a float. I had also assumed is was integral, and my original code did and all-integer computation (see comments above), but it turned out to be a float, so I switched to `std::ceil()` instead.

Maybe I should make wanted_range integral, while I'm at it ?